QBImagePicker/QBCheckmarkView.h

```diff
@@ -18,4 +18,6 @@ IB_DESIGNABLE
 @property (nonatomic, strong) IBInspectable UIColor *bodyColor;
 @property (nonatomic, strong) IBInspectable UIColor *checkmarkColor;
 
+@property (nonatomic, weak) IBOutlet UILabel *label;
```

Comment:
I'm going to assume that `nonatomic, weak` is the standard annotation for outlets added by Interface Builder? Sorta noideadog on this.

Reply:
Right, when I connected the magic dots in the storyboard UI this was the code that was generated.